CoreScriptsRoot/Modules/Chat.lua

```diff
@@ -1946,18 +1947,14 @@ local function CreateChatWindowWidget(settings)
 				{
 					Name = 'MessageContainer';
 					Size = UDim2.new(1, -SCROLLBAR_THICKNESS - 1, 0, 0);
-					Position = UDim2.new(0, 0, 1, 0);
+					Position = UDim2.new(0, 0, 0, 0);
 					ZIndex = 1;
 					BackgroundColor3 = Color3.new(0, 0, 0);
 					BackgroundTransparency = 1;
 					Parent = scrollingFrame
 				};
 
-		-- This is some trickery we are doing to make the first chat messages appear at the bottom and go towards the top.
 		local function OnChatWindowResize(prop)
-			if prop == 'AbsoluteSize' then
```

Comment:
This also was for adjusting the chat to fill in from the bottom up, it won't be needed if we're starting from the top.